easybuild/easyconfigs/x/X11/X11-20200222-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,208 @@
+easyblock = 'Bundle'
+
+name = 'X11'
+version = '20200222'
+
+homepage = 'https://www.x.org'
+description = "The X Window System (X11) is a windowing system for bitmap displays"
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = [
+    XORG_LIB_SOURCE,
+    XORG_PROTO_SOURCE,
+    'https://xcb.freedesktop.org/dist/',
+    'https://xkbcommon.org/download/',
+    XORG_DATA_SOURCE + '/xkeyboard-config',
+    XORG_DATA_SOURCE,
+]
+
+builddependencies = [
+    ('binutils', '2.34'),
+    ('Autotools', '20180311'),
+    ('Bison', '3.5.3'),
+    ('gettext', '0.20.1'),
+    ('pkg-config', '0.29.2'),
+    ('intltool', '0.51.0'),
+    ('Meson', '0.53.2', '-Python-3.8.2'),
+    ('Ninja', '1.10.0'),
+]
+dependencies = [
+    ('bzip2', '1.0.8'),
+    ('fontconfig', '2.13.92'),
+    ('freetype', '2.10.1'),
+    ('zlib', '1.2.11'),
+    ('xorg-macros', '1.19.2'),
+    ('libpciaccess', '0.16'),
+]
+
+default_easyblock = 'ConfigureMake'
+
+default_component_specs = {
+    'sources': [SOURCE_TAR_GZ],
+    'start_dir': '%(name)s-%(version)s',
+}
+components = [
+    ('libpthread-stubs', '0.4', {  # 2017-03-14
+        'checksums': ['50d5686b79019ccea08bcbd7b02fe5a40634abcfd4146b6e75c6420cc170e9d9'],
+    }),
+    ('xorgproto', '2019.2', {  # 2019-10-17
+        'checksums': ['ebfcfce48b66bec25d5dff0e9510e04053ef78e51a8eabeeee4c00e399226d61'],
+    }),
+    ('libXau', '1.0.9', {  # 2019-02-10
+        'checksums': ['1f123d8304b082ad63a9e89376400a3b1d4c29e67e3ea07b3f659cccca690eea'],
+    }),
+    ('libXdmcp', '1.1.3', {  # 2019-03-16
+        'checksums': ['2ef9653d32e09d1bf1b837d0e0311024979653fe755ad3aaada8db1aa6ea180c'],
+    }),
+    ('xcb-proto', '1.14', {  # 2020-02-22
+        'checksums': ['1c3fa23d091fb5e4f1e9bf145a902161cec00d260fabf880a7a248b02ab27031'],
+    }),
+    ('libxcb', '1.14', {  # 2020-02-22
+        'sources': [SOURCE_TAR_GZ],
+        'checksums': ['2c7fcddd1da34d9b238c9caeda20d3bd7486456fc50b3cc6567185dbd5b0ad02'],
+    }),
+    ('xtrans', '1.4.0', {  # 2019-03-16
+        'checksums': ['48ed850ce772fef1b44ca23639b0a57e38884045ed2cbb18ab137ef33ec713f9'],
+    }),
+    ('libxkbcommon', '0.10.0', {  # 2020-01-18
+        'easyblock': 'MesonNinja',
+        'sources': ['libxkbcommon-%(version)s.tar.xz'],
+        'checksums': ['57c3630cdc38fb4734cd57fa349e92244f5ae3862813e533cedbd86721a0b6f2'],
+        'preconfigopts': '',
+        'configopts': '-Denable-wayland=false ',
```

Comment:
You either need to introduce a doxygen dep or add `-Denable-docs=false` to the `configopts`

Reply:
Updated